manifests/g/Google/AndroidStudio/2020.3/Google.AndroidStudio.locale.en-US.yaml

```
@@ -1,22 +1,19 @@
-# Created using YamlCreate.ps1 v1.3.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
-
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultlocale.1.0.0.schema.json
 PackageIdentifier: Google.AndroidStudio
-PackageVersion: 2020.3
+PackageVersion: "2020.3"
 PackageLocale: en-US
 Publisher: Google LLC
 PublisherUrl: https://developer.android.com/studio
 PublisherSupportUrl: https://developer.android.com/studio/intro
 PrivacyUrl: https://policies.google.com/privacy
-#Author: 
 PackageName: Android Studio
 PackageUrl: https://developer.android.com/studio#downloads
 License: Android Software Development Kit License Agreement
 LicenseUrl: https://developer.android.com/studio/terms
 Copyright: Copyright (c) Google LLC.
 CopyrightUrl: https://developer.android.com/studio#downloads
 ShortDescription: Android Studio provides the fastest tools for building apps on every type of Android device.
-Description: Android Studio is Android's official IDE. It is purpose-built for Android to accelerate your development and help you build the highest-quality apps for every Android device.
+Description: Android Studio is Android"s official IDE. It is purpose-built for Android to accelerate your development and help you build the highest-quality apps for every Android device.
```

```suggestion
Description: Android Studio is Android's official IDE. It is purpose-built for Android to accelerate your development and help you build the highest-quality apps for every Android device.
```
I think this is supposed to be `'`?

---

Whoops. Sorry about that.